components/content/repulsion-effect/index.vue

```diff
@@ -0,0 +1,133 @@
+<script setup lang="ts">
+import { TresCanvas, useTresContext } from '@tresjs/core'
+import { PCFSoftShadowMap, BasicShadowMap, PCFShadowMap, NoColorSpace, LinearSRGBColorSpace, SRGBColorSpace, NoToneMapping } from 'three'
+import { OrbitControls, Box, vAlwaysLookAt, vLightHelper } from '@tresjs/cientos';
```

Comment:
Do you need to import them?
I think we have auto-import from nuxt here 

Also, some imports are not been utilized

Reply:
Exactly! Small error on my part!